src/blockchain/node/base/Base.cpp

```diff
@@ -1186,25 +1180,12 @@ auto Base::SendToPaymentCode(
     return SendToPaymentCode(nymID, recipient.asBase58(), amount, memo);
 }
 
-auto Base::shutdown(std::promise<void>& promise) noexcept -> void
+auto Base::shut_down() noexcept -> void
```

Comment:
why such name with undescore ? it oveerides sth different with such name ?

Reply:
'shutdown' is used a lot in other cases. Besides, I always try to use function names that include a verb, that makes the code a bit more readable.